src/components/MovieContainer/MovieContainer.ts

```diff
@@ -1,13 +1,19 @@
-import { addShowMoreButtonEventListener } from '../ShowMoreButton/eventHandler';
 import { renderHandler } from './render';
-import loadingOrErrorStateUIManager from '../../services/LoadingOrErrorStateUIManager';
 import { API_ENDPOINT, API_OPTION } from '../../constants/api/api';
+import renderMovieItemDetailModalHandler from '../MovieItem/eventHandler';
+import { initializeInfiniteScroll } from '../ShowMoreButton/infiniteScrollHandler';
+import movieFetcherWithLoadingOrErrorState from '../../services/MovieFetcherWithLoadingOrErrorState';
+import MovieStorageService from '../../services/MovieStorageService';
 
 async function MovieContainer() {
-  const data = await loadingOrErrorStateUIManager.fetchData(API_ENDPOINT.POPULAR(), { headers: API_OPTION.headers });
-  const { results } = data;
-  renderHandler(results);
-  addShowMoreButtonEventListener();
+  const dataFromServer = await movieFetcherWithLoadingOrErrorState.fetchData(API_ENDPOINT.POPULAR(), {
+    headers: API_OPTION.headers,
+  });
+  const { results } = dataFromServer;
+  const data = MovieStorageService.addData(results);
+  renderHandler(data);
+  initializeInfiniteScroll();
+  renderMovieItemDetailModalHandler();
```

Comment:
```suggestion
  const dataFromServer = await movieFetcherWithLoadingOrErrorState.fetchData(API_ENDPOINT.POPULAR(), {
    headers: API_OPTION.headers,
  });
  const { results } = dataFromServer;
  const data = MovieStorageService.addData(results);
  
  renderHandler(data);
  initializeInfiniteScroll();
  renderMovieItemDetailModalHandler();
```

선언부 밑에 개행이 있으면 조금 숨 쉴 여유가 있을 것 같아요!
대체로 급하게 제출해서 그런지 일관성이 지켜지지 못한 부분들이 더러 있네요 ㅜㅜ

Reply:
> 대체로 급하게 제출해서 그런지 일관성이 지켜지지 못한 부분들이 더러 있네요 ㅜㅜ
위와 관련된 표현은 사과를 드려야 겠습니다.
“버건디가 급하게 제출했다“로 단정지은 듯한 말투로 오해를 사게 됐네요.

### 급하게 제출했다는 의미
- 기능 요구사항을 충족하지 못했는데 PR부터 올렸다 (X)
- 1등으로 PR을 올려야 하니 급한 마음으로 검토를 소홀히하고 PR을 올렸다 (X)
- "조금 더 고민하면서 다듬을 시간이 필요했는데 일신상의 사유로 (크론이 생각하기에) 아쉬운 상태에서 PR을 올렸다"가 타당할 것 같아요. 조금 더 상대방의 입장에서 이 표현이 어떻게 받아들여질지 고민했어야 했는데 저야말로 급하게 리뷰를 단 것 같아 죄송한 마음입니다. 본의 아니게 버건디의 마음가짐에 대해서도 알게 됐지만, 저는 버건디가 1등으로 PR을 올려야 한다거나 기능 요구사항을 충족하지 못했지만 일단 PR부터 올렸다고 생각하는 것은 아니었어요. 위에서 말씀해 주신 대로 지금 시점에 PR을 올리고 리뷰로 많은 포인트를 챙기면서 개선해 나가는 과정을 제가 강조하고 있기 때문이기도 하고요.

### 코드 리뷰와 피드백이라는 의미

&nbsp;&nbsp;저도 처음에는 "피드백"과 "리뷰"는 나에 대한 부족한 부분을 "지적한다"는 의미로 받아들였었는데요. 그게 아닌 것 같더라구요. 제가 드리는 피드백도 "지적"이 아니라 버건디와 다른 관점에서 고민해 봤는데 이건 어떠신가요? 라고 하는 제안 정도로 받아들여 주세요. 마치 UDP처럼요! 아무래도 버건디는 현직 경험이 없지만, 코치나 리뷰어는 현직 경험이 있으시니까 리뷰를 받을 때마다 남다른 "무게감"이 느껴지실 수도 있을 것 같은데요. 리뷰어나 저의 입장에서는 사실 피드백은 "제안"에 가깝습니다(정말 꼭 고쳐야 할 것 같은 코드를 포인트로 짚었다고 할지라도요). 

&nbsp;&nbsp;리뷰어님들은 버건디에게 코드 리뷰를 통해 피드백을 할 수 있는 포인트를 찾고 그것을 통해 버건디가 개선해 나가며 성장하는 것을 보면서 보람을 느끼실 겁니다. 저도 피드백을 통해서 버건디가 코드를 개선해 나가며 성장하는 모습에 희열을 느끼고 싶었어요. 그런데 이건 제가 정말 고쳐야 하는 것이지만, 제가 욕심이 많습니다😓. 리뷰 포인트를 짚어주면서 보람을 느낌과 동시에 여기에서 아쉬운 점이 보이면 제 마음이 좀 더 다급해지는 것도 있는 것 같아요. 이러한 저의 욕심이 한 문장으로 드러났는데 이것이 버건디의 심기를 불편하게 만들었다면 다시 한 번 더 사과드리겠습니다🙏. 어쩌면 미션 포인트를 잘 지켰다고 이야기해드리는 긍정적인 피드백을 드리는 것이 크루들에게 도움이 될 수도 있을 텐데 앞선 욕심 때문에 항상 피드백 포인트를 짚어 드리다가 시간 관계상 긍정적인 리뷰 포인트는 자연스럽게 스킵하게 되는 것 같아요. (~크루들에게 욕심을 부리지 말라고 하면서 저는 욕심을 부리고 있네요...~)

그리고 버건디도 충분히 열심히 하고 있다는 것을 느끼고 있습니다 🙂
제가 급한 마음을 좀 억누르고 욕심을 자제해보겠습니다.
그러니까 앞으로의 미션도 영화리뷰처럼 해주시면 좋을 것 같습니다!

Reply:
두 분 말씀 읽으면서 크론도, 버건디도 각자의 역할을 너무 멋있게 해주고 계신 것 같아 감동 받았습니다 😲 

저도 코치님과 크루분들께 도움이 되도록 성실히 리뷰 남기겠습니다! 😊